src/test/index.ts

```diff
@@ -1,10 +1,17 @@
+import { getRepository } from 'typeorm';
+import { generateSeed } from '../seed';
 import { setup } from '../setup';
 
 before(async () => {
+  await generateSeed();
   await setup();
 });
+after(async () => {
+  await getRepository('User').clear();
+});
 
 // require('./hello-world.test');
 // require('./create-user.test');
-require('./user-query.test');
+// require('./user-query.test');
 // require('./login.test');
```

Comment:
Por que os outros testes estão comentados? Um dos maiores benefícios dos testes é que eles continuem rodando e continuem validando as outras queries/mutations enquanto o código vai sendo modificado.

De repente alguma feature trigga algum refactor ou melhoria de código, e você acaba mexendo numa parte de outra query/mutation. Aí vc tem os testes pra te salvar e dizer se continua tudo bem nelas.

Reply:
vou tirar o comentário! eu comentei pra ver especificamente o que eu tava fazendo hahaha

Reply:
@dan-ueno 
Vc pode usar a opção de grep do mocha (-g):

```bash
npm run test -- -g "user query"
```

Ele roda apenas os `describe`s ou `it`s que passarem no filtro do -g